widget/embedded/src/store/slices/data.ts

```diff
@@ -48,8 +114,11 @@ export const createDataSlice: StateCreator<
   DataSlice
 > = (set, get) => ({
   // State
-  _blockchainsMapByName: new Map<string, BlockchainMeta>(),
-  _tokensMapByTokenHash: new Map<TokenHash, Token>(),
+  _blockchainsMapByName: new Map(),
+  _tokensMapByTokenHash: new Map(),
+  _tokensMapByBlockchainName: {},
+  _supportedSourceTokens: [],
```

Comment:
Why do we need `_supportedSourceTokens` anymore? and also `_supportedDestinationTokens`.